Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.13.8
```

Comment:
On the release file, version 1.15 is used. Use also 1.15 here ?

Reply:
Updated to 1.15